core/arch/arm/include/mm/core_mmu_arch.h

```diff
@@ -210,6 +210,12 @@ static inline unsigned int core_mmu_get_va_width(void)
 	}
 	return 32;
 }
+
+static inline bool core_mmu_table_level_not_exceed(unsigned int level)
```

Comment:
replicate this in _core/arch/riscv/include/mm/core_mmu_arch.h_

Reply:
Hi @etienne-lms 

RISC-V uses decreasing style in address translation level, which is different with ARM.
And since new RISC-V mm implementation https://github.com/OP-TEE/optee_os/pull/6104 is not ready to be merged, I hope RISC-V's version can be implemented in another PR. Some table level definitions are changed in #6104.

Reply:
riscv builds _boot.c_ that now requires `core_mmu_table_level_not_exceed()`.

Reply:
I add a new commit for RISC-V's `core_mmu_table_level_not_exceed()`.
Please check it.